src/background/hypothesis-chrome-extension.js

```diff
@@ -35,24 +35,19 @@ import { TabStore } from './tab-store';
  * @param {chrome.storage} services.chromeStorage
  * @param {chrome.browserAction} services.chromeBrowserAction
  * @param {(path: string) => string} services.extensionURL
- * @param {(cb: (allowed: boolean) => void) => void} services.isAllowedFileSchemeAccess
  */
 export default function HypothesisChromeExtension({
   chromeTabs,
   chromeExtension,
   chromeStorage,
   chromeBrowserAction,
   extensionURL,
-  isAllowedFileSchemeAccess,
```

Comment:
Testing my understanding: Will more of these parameters go away as you implement the new API in the other modules (`HelpPage`, `TabStore`, etc.)?

Reply:
Yes. All of these will eventually go away.